test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -514,46 +749,62 @@ test.requestHooks(logger, enrollViaSmsVersionUpgradeMocksGoBack)('should not sho
   await t.expect(enrollViaSMSPageObject.hasCountryField()).ok();
   await t.expect(enrollViaSMSPageObject.getCountryLabel()).eql('+1');
   await enrollViaSMSPageObject.fillPhoneField('8887227871');
-  await enrollViaSMSPageObject.clickNextButton();
-  await t.expect(logger.count(() => true)).eql(3);
-  const { request: { body: answerRequestBodyString }} = logger.requests[2];
-  const answerRequestBody = JSON.parse(answerRequestBodyString);
-  await t.expect(answerRequestBody.phoneNumber).eql('+18887227871');
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(false);
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaEmailInstruction()).eql(false);
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaSmsInstruction()).eql(true);
-  await t.expect(enrollOktaVerifyPage.getSmsInstruction()).eql(smsInstruction);
+  await enrollViaSMSPageObject.clickSendSetupLink();
+  // logger.count ~ 10 in v3, and not consistent
+  if (!userVariables.v3) {
+    await t.expect(logger.count(() => true)).eql(3);
+    const { request: { body: answerRequestBodyString }} = logger.requests[2];
+    const answerRequestBody = JSON.parse(answerRequestBodyString);
+    await t.expect(answerRequestBody.phoneNumber).eql('+18887227871');
+  }
+  
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(false);
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaEmailInstruction()).eql(false);
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaSmsInstruction()).eql(true);
+  await t.expect(await enrollOktaVerifyPage.getSmsInstruction()).contains(smsInstruction1);
+  await t.expect(await enrollOktaVerifyPage.getSmsInstruction()).contains(smsInstruction2);  
+  
   const errorBox = enrollOktaVerifyPage.getErrorBox();
   await t.expect(errorBox.innerText).contains(fipsUpgradeMessage);
   const errorTitle = enrollOktaVerifyPage.getErrorTitle();
   await t.expect(errorTitle.innerText).contains(fipsUpgradeTitle);
   // hit go back
   await enrollOktaVerifyPage.switchAuthenticator();
-  await t.expect(errorBox.exists).notOk;
+  t.expect(errorBox.exists).notOk;
```

Comment:
was the 'await' not necessary here?

Reply:
It wasn't. I got an info message saying await doesn't have any effect here.